ts/features/fims/history/saga/index.ts

```diff
@@ -0,0 +1,16 @@
+import { takeLatest } from "typed-redux-saga/macro";
+import { FimsHistoryClient } from "../api/client";
+import { fimsHistoryGet } from "../store/actions";
+import { handleGetFimsHistorySaga } from "./handleGetFimsHistorySaga";
+
+export function* watchFimsHistorySaga(
+  client: FimsHistoryClient,
+  bearerToken: string
+) {
+  yield* takeLatest(
```

Comment:
Do we really need this nested step? Why not merge it directly into `fims/common/saga/index`, avoid a two-level depth into the sagas?

Reply:
in this exact moment yes, but in a couple PRs (aka when the export will be implemented), I was planning to add it in the history subfeature, so this will handle that.